Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CameraHelper/CameraHelperCode.bind

```diff
@@ -21,7 +21,4 @@ private void CameraHelper_FrameArrived(object sender, FrameEventArgs e)
 {
   // Gets the current video frame
   VideoFrame currentVideoFrame  = e.VideoFrame;
-
-  // Gets the software bitmap image
-  SoftwareBitmap softwareBitmap = e.SoftwareBitmap;
```

Comment:
This change is not necessary, I have updated it to get the software bitmap from video frame

Reply:
by not necessary you mean what? you already changed the .bind?